.changeset/chilled-cooks-listen.md

```diff
@@ -0,0 +1,5 @@
+---
+"hardhat": patch
+---
+
+Add `personal_sign` RPC method
```

Comment:
```suggestion
Add support for the `personal_sign RPC method to Hardhat Network.
```

I think we should write them as we would for the release. wdyt?

Reply:
Yes, makes sense!